requirements.txt

```diff
@@ -1,5 +1,4 @@
 h5py==2.10.0
 hdmf==2.1.0
 numpy==1.18.5
-pandas==0.25.3
```

Comment:
pandas is still a requirement though, right?

Reply:
My understanding is that having pandas as a requirement for install should not be an issue. If we "just" want to disable certain functionality to speed-up import, then an alternative way would be to use environment variables such that, e.g., if PYNWB_NOPANDAS is set then you would not import Pandas, i.e., the user would have to explicitly set the variable before importing pynwb. Having a special "pynwb-lite" may be the "cleaner" variant but I'm afraid that this will ultimately add a lot of work and I don't think we have sufficient resources right now to support such an effort.